codegen/smithy-aws-python-codegen/src/main/java/software/amazon/smithy/aws/python/codegen/AwsAuthIntegration.java

```diff
@@ -1,16 +0,0 @@
-/*
```

Comment:
I get that these do nothing right now, but they were there so that the Python team didn't have to set up all this, so I'd appreciate you keeping them around in whatever package name aligns with what sj is doing